### DISCUSSION TITLE
Addressing Concerns and Proposing Solutions for Spacemesh Network Health and Decentralization

### DISCUSSION THREAD

**Discussion by bugxzhu** [spacemeshos/go-spacemesh#5104] [Ideas]
Dear Spacemesh Team,

I would like to share some concerns and suggestions regarding the Spacemesh network. First and foremost, I want to express my support and interest in the Spacemesh project. Your work and vision in the blockchain space have garnered widespread attention. However, there are some issues of concern at present, particularly related to pool control and the reward mechanism.

In the current Spacemesh network, it appears that the H9 pool has gained control of a significant portion of the hashing power. This raises concerns as a very small number of nodes within the H9 pool seem to be able to control a large portion of the network's hashing power, posing a threat to the project's health and decentralization. This situation is not only unfriendly to newcomers but also poses centralization risks.

Another issue pertains to the reward mechanism. Currently, miners in the H9 pool can stop their nodes immediately after completing a Post and still remain eligible for rewards. In contrast, solo miners are expected to keep their nodes online continuously, which is considered unfair to some extent. This could lead to more people joining the H9 pool, which goes against the original decentralized vision of the Spacemesh project.

Lastly, concerns related to miners being able to continue receiving rewards after shutting down their nodes may negatively impact the price stability of the Spacemesh token. Because miners can easily shut down their nodes, this results in a very low cost for earning rewards, potentially causing instability in the supply of rewards and affecting the market price of the token.

Therefore, I strongly urge the Spacemesh team to consider introducing mechanisms to combat pool dominance and ensure the network's health and decentralization are maintained. This may involve limiting the hashing power share of individual pools or implementing other technical measures to mitigate centralization risks. Additionally, a review of the reward mechanism to ensure fair miner participation consistent with the project's vision is recommended.

Once again, thank you for your work and contributions to the blockchain space. I look forward to seeing the continued success of the Spacemesh project while preserving its decentralized and community-friendly characteristics.

Best regards

**Comment by lrettig** [comment]
Hi @bugxzhu , thanks for raising these concerns. Please believe me when I say that decentralization is both one of our core values and also very front of mind for us as we continue to improve the Spacemesh network and protocol. I think your concerns are valid, but at the same time, we have to recognize that decentralization is more of a long term goal that we work towards and less of a thing that we can quickly modify the protocol to improve in the short term. As the Spacemesh protocol and network continue to mature, we're confident that it will get easier to run a node at home and decentralization will continue to increase. We're hard at work on many aspects of these improvements and we'll share more soon on what we're working on and the impact we expect these improvements to have. I don't think there are any quick, short-term fixes to the issues you raise. If there were, we would already have implemented them.

That being said, Spacemesh is of course an open project and you and other community members are welcome to propose ideas and protocol changes and improvements, informally here or more formally as [SMIPs](https://github.com/smfoundation/smips). Are there any concrete ideas you'd like to propose or discuss here?

Thanks!
